src/Scaler.Tests/CosmosDbScalerServiceTests.cs

```diff
@@ -18,10 +18,10 @@ public CosmosDbScalerServiceTests()
         }
 
         [Theory]
-        [InlineData("connection")]
+        [InlineData("endpoint")]
```

Comment:
Retain support for connection-string based connections and have tests for both scenarios.